package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@bigcommerce/handlebars-v4": "4.7.6",
     "handlebars": "3.0.8",
-    "handlebars-helpers": "0.8.4",
+    "handlebars-helpers": "0.10.0",
```

Comment:
handlebars-helpers doesn't follow semver and introduced some breaking changes -- check their changelog

Reply:
It is mostly addition. I am going through the ones which were changed to confirm we are not using them